cmd/context/use.go

```diff
@@ -137,7 +164,17 @@ func (c *Command) Run(ctx context.Context, ctxOptions *Options) error {
 		ctxOptions.Save = true
 	}
 
+	if err := c.forceLoginIfRequested(ctxOptions, ctxStore); err != nil {
+		return err
+	}
+
 	if err := c.UseContext(ctx, ctxOptions); err != nil {
+		// delete the context to force the user to log in again
+		if oktetoErrors.IsNotFound(err) {
```

Comment:
this is handling the specific error we saw on the replicated migration:
```
 x  Non-200 OK status code: 401 Unauthorized body: "fail to find user with token XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx309: not-found\n"
```

This happens when you don't have a valid token on your context file. I think we should fix this at the root, but I don't want the agent to do that big of a refactoring.